### ISSUE TITLE
Wrong arrow explanation for vaccination stats

### ISSUE THREAD

**Issue by MDXDave** [corona-warn-app/cwa-documentation#659]
The vaccination stats shows a red arrow that points downwards
![Screenshot_20210714-130728](https://user-images.githubusercontent.com/6099976/125612799-52b01915-a70d-4028-a6e1-8f3f42d9000d.jpg)

But the explanation says that red arrows indicates an increasement.
![Screenshot_20210714-130723~2](https://user-images.githubusercontent.com/6099976/125612796-8801b62c-bfda-416e-8a17-06685a2828ca.jpg)

The text or the background color of the arrow should be adjusted or added to the explanation. 

---
Internal Tracking ID: [EXPOSUREAPP-8403](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8403)

**Comment by dsarkar** [comment]
@Ein-Tim Thanks. Internal Tracking ID: [EXPOSUREAPP-8403](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8403)

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
FYI: PRs:

- Android: https://github.com/corona-warn-app/cwa-app-android/pull/3726
- iOS: https://github.com/corona-warn-app/cwa-app-ios/pull/3168

**Comment by MikeMcC399** [comment]
In German in Android through https://github.com/corona-warn-app/cwa-app-android/pull/3726 for CWA 2.6 the following changes were implemented for CWA 2.6:

https://github.com/corona-warn-app/cwa-app-android/blob/fc7c1bd7377a9db04cb50abc06588c1d93400041/Corona-Warn-App/src/main/res/values-de/strings.xml#L1475-L1481
```
    <string name="statistics_explanation_trend_icons_title">"Beispiele für die angezeigten Trends:"</string>
    <!-- XHED: Explanation screen trend increasing title -->
    <string name="statistics_explanation_trend_increasing_title">"Trend steigend, negativ"</string>
    <!-- XHED: Explanation screen trend decreasing title -->
    <string name="statistics_explanation_trend_decreasing_title">"Trend fallend, positiv"</string>
    <!-- XHED: Explanation screen trend stable title -->
    <string name="statistics_explanation_trend_stable_title">"Trend stabil, neutral"</string>
```

In English
https://github.com/corona-warn-app/cwa-app-android/blob/fc7c1bd7377a9db04cb50abc06588c1d93400041/Corona-Warn-App/src/main/res/values/strings.xml#L1474-L1480
```
    <string name="statistics_explanation_trend_icons_title">"The following trends can be shown:"</string>
    <!-- XHED: Explanation screen trend increasing title -->
    <string name="statistics_explanation_trend_increasing_title">"Trend: upwards (negative)"</string>
    <!-- XHED: Explanation screen trend decreasing title -->
    <string name="statistics_explanation_trend_decreasing_title">"Trend: downwards (positive)"</string>
    <!-- XHED: Explanation screen trend stable title -->
    <string name="statistics_explanation_trend_stable_title">"Trend: steady (neutral)"</string>
```

The translation to English would be better, if it made clear that the icons and colours shown below are examples and not the only combinations which can be shown.
`    <string name="statistics_explanation_trend_icons_title">"The following trends can be shown:"</string>`

---
I suggest changing `statistics_explanation_trend_icons_title` to
"The following are examples of trends shown:"
in the English translation.

At least the word "examples" should appear in the text to correspond to the word "Beispiele" in German.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks. We will forward this to the internal ticket.

**Comment by MikeMcC399** [comment]
@dsarkar 
There hasn't been any change made to `statistics_explanation_trend_icons_title` yet.

Should this issue be closed and a new issue be opened to follow the suggestion to modify the English translation of  `statistics_explanation_trend_icons_title` as described in https://github.com/corona-warn-app/cwa-documentation/issues/659#issuecomment-889061159?

If this issue is left open, then probably the 2.6 fix label should be removed, otherwise the open status of the issue, with a fix label for a version which has already been released, is confusing.

**Comment by MikeMcC399** [comment]
@dsarkar 
For clarity I described the remaining problem from this issue (#659) in a new issue ("Confusing English explanation for trend icons" #757).

I suggest that this issue (#659) can now be closed.

**Comment by MikeMcC399** [comment]
This issue should be closed now as the underlying issue was resolved in release 2.15.

- #757 has been resolved and is closed.

The current CWA Android 2.17.2 app displays the following where it is clarified that a red background behind a trend arrow is a negative trend. The picture of a red arrow trending upwards is described as an example only. 

So it should now be clear that a trend arrow can be up or down and have the negative red colour associated with it.

(It is also fixed in the English-language display.)

![Verabreichte Impfdosen](https://user-images.githubusercontent.com/66998419/154099679-f4d9886d-f55a-4928-9fce-f55a01330cce.png)

![Kennzahlen Trend](https://user-images.githubusercontent.com/66998419/154099736-461f890e-438f-4830-abd9-cc2d7f6488fb.png)

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for CONFIRMING the fix.
@MDXDave Thanks for reporting this issue. The explanation has been adjusted. We are closing this issue now as fixed.

Thanks everybody!


Best wishes, DS

---
Corona-Warn-App Open Source Team
